<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.kodilla.spring.reader.ReaderTestSuite" tests="2" skipped="0" failures="0" errors="0" timestamp="2019-10-17T13:46:32" hostname="PL-08818-L2979" time="0.059">
  <properties/>
  <testcase name="testRead" classname="com.kodilla.spring.reader.ReaderTestSuite" time="0.033"/>
  <testcase name="testConditional" classname="com.kodilla.spring.reader.ReaderTestSuite" time="0.026"/>
  <system-out><![CDATA[2019-10-17 15:46:32.456  INFO 140 --- [    Test worker] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@6bb58520: startup date [Thu Oct 17 15:46:32 CEST 2019]; root of context hierarchy
Reading: The book number one
2019-10-17 15:46:32.487  INFO 140 --- [    Test worker] s.c.a.AnnotationConfigApplicationContext : Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@5a0316db: startup date [Thu Oct 17 15:46:32 CEST 2019]; root of context hierarchy
Bean book2 was found in the container: true
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
